www/includes/postcode.inc

```diff
@@ -17,7 +17,7 @@ function _postcode_to_constituencies($postcode, $mp_only) {
     $postcode = canonicalise_postcode($postcode);
 
     if ($last_postcode == $postcode) {
-        $return_value = $mp_only ? $last_postcode_value['WMC'] : $last_postcode_value;
+        $return_value = is_array($last_postcode_value) ? $last_postcode_value['WMC'] : $last_postcode_value;
```

Comment:
last_postcode_value is always an array, so this won't be returning the right value when mp_only is false, I think? Not sure what the actual issue is here, sorry; is something calling this function with an incorrect second argument, perhaps?